subsys/nrf_security/src/drivers/cracen/cracenpsa/include/cracen_psa_key_ids.h

```diff
@@ -17,4 +17,8 @@
 
 #define PSA_KEY_LOCATION_CRACEN ((psa_key_location_t)(0x800000 | ('N' << 8)))
 
+/* Persistence state were deleted keys are revoked, and the key slot can not be provisioned again.
```

Comment:
something is off with the grammar of the comment here, no? 

"Persistence state were deleted keys are revoked"

Is the issue that were ->where or there is something else that you want to say?